src/components/button/button.config.ts

```diff
@@ -7,5 +11,5 @@ export const BUTTON_VARIANTS: ButtonTypes[] = [
   "dashed",
   "darkBackground",
 ];
-export const BUTTON_SIZES = ["small", "medium", "large"];
-export const BUTTON_ICON_POSITIONS = ["before", "after"];
+export const BUTTON_SIZES: SizeOptions[] = ["small", "medium", "large"];
```

Comment:
Why was this and BUTTON_ICON_POSITIONS below changed to this format with the added `: SizeOptions[]`?

Reply:
provided specific Type for it